packages/mobile/src/exchange/ExchangeScreenHeader.tsx

```diff
@@ -43,9 +42,9 @@ function ExchangeTradeScreenHeader({ currency, makerToken, onChangeCurrency }: P
 
     let titleText
     let title
-    const singleTokenAvailable = currenciesWithBalance < 2
-    if (singleTokenAvailable) {
-      title = isCeloPurchase ? i18n.t('exchangeFlow9:buyGold') : i18n.t('exchangeFlow9:sellGold')
+    const tokenPickerEnabled = currenciesWithBalance >= 2 || !isCeloPurchase
+    if (!tokenPickerEnabled) {
+      title = i18n.t('exchangeFlow9:buyGold')
```

Comment:
I'm a little confused by this...is there never a time where `title` should equal `exchangeFlow9:buyGold`?

Reply:
For selling CELO we always show both currencies no matter what, and when we show both currencies we don't show the buy/sell CELO text, the header shows up like this:

<img width="395" alt="Screen Shot 2021-07-12 at 15 07 10" src="https://user-images.githubusercontent.com/6062888/125335358-d660e680-e322-11eb-9695-03cb5fcd68ca.png">

so in no cases does the title say `exchangeFlow9:sellGold` and it only says `exchangeFlow9:buyGold` when the user is buying CELO and they have a balance in only one currency.

Reply:
Got it. Can we delete `exchangeFlow9:buyGold`?

Reply:
It's still used in the header of the `ExchangeReviewScreen`